pkg/lib/queueinformer/queueinformer_operator.go

```diff
@@ -144,6 +144,7 @@ func (o *operator) addHasSynced(hasSynced cache.InformerSynced) {
 	}
 }
 
+// TODO: generalize over func(ctx) so this can start informers + source watcher
```

Comment:
Alternatively, you could create a method on the catalog `Operator` that starts the source watcher after/before calling this.

Reply:
that's essentially how it's configured now - did you have something else in mind?

Reply:
No - just making 2 calls -> 1 from from the `catalog.Operator` client perspective.